UnitTests/Tests/Controllers/TRApplicationControllerSpec.m

```diff
@@ -61,6 +62,44 @@ @interface TRWeatherController (Tests)
             OCMVerify([updateWeatherCommand execute:applicationController]);
         });
     });
+
+    describe(@"subscribeForNotificationsWithDeviceToken:", ^{
+        it(@"subscribes to notifications on the channel corresponding to the device's time zone", ^{
+            TRCourierClient *courier = OCMClassMock([TRCourierClient class]);
+            TRApplicationController *applicationController = [TRApplicationController new];
+            applicationController.courier = courier;
+
+            NSData *deviceToken = [NSData new];
+            [applicationController subscribeForNotificationsWithDeviceToken:deviceToken];
+
+            NSString *channel = [[NSTimeZone localTimeZone] name];
```

Comment:
What do you think about renaming this variable to `expectedChannel`, and moving its definition into the "arrange" portion of the test before the call to `subscribeForNotificationsWithDeviceToken:`?